db/pagination/pagination.go

```diff
@@ -35,15 +35,16 @@ const (
 
 //Paginator stores pagination data
 type Paginator struct {
-	Key    string
-	Order  string
-	Limit  uint64
-	Offset uint64
-	Fields []string
+	Key     string
+	Order   string
+	Limit   uint64
+	Offset  uint64
+	Fields  []string
```

Comment:
This doesn't belong here. Paginator has very specific purpose and resource details ins't it.